.github/workflows/tests.partners.on_push.yaml

```diff
@@ -34,6 +34,8 @@ jobs:
     steps:
       - name: Check out repository code
         uses: actions/checkout@v2
```

Comment:
we might bump this to `actions/checkout@v3`
https://github.com/actions/checkout/compare/releases/v2...main

Reply:
bumped. tests running once more and then I'll merge.